tests/dummy/app/templates/docs/scroll-into-view.md

```diff
@@ -19,6 +19,29 @@ You should use this modifier whenever you need to have an element scrolled into
 `shouldScroll` can be either a Boolean or a Promise that resolves to a truthy or falsy value. It does not handle a rejected Promise.
 
 
+## Testing
+`scroll-into-view-mock` provides a function that will mock the native browser `scrollIntoView` and allow testing which elements invoked the modifier
+
+`mockScrollIntoView()` - will mock the native API and return an object with the following 2 functions
+* `scrollIntoViewCalledWith(Element|DOM selector)` - tests if the modifier was invoked on the element
+* `resetMock()` - returns the native scrollIntoView function
+
+```javascript
+import mockScrollIntoView from 'ember-scroll-modifiers/test-support/scroll-into-view-mock';
+
+...
+const mockHelperFunctions = mockScrollIntoView();
+...
+await render(
+  hbs`<div {{scroll-into-view shouldScroll=true}} data-test-scroll-into-view-selector></div><div data-test-scroll-into-view-not-modified></div><div {{scroll-into-view shouldScroll=false}} data-test-scroll-into-view-not-scrolled></div>`
```

Comment:
I don't think we need to include the extra divs here - people will be confused by them. Let's just have 1 div with the modifier and data-test selector.